web/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.7'
```

Comment:
How about having just one docker-compose file in the main folder?

Reply:
Do you mean for all services "web", "alerting", and "api-flask"?